vpr/src/pack/post_routing_pb_pin_fixup.cpp

```diff
@@ -581,6 +587,19 @@ static void rec_remove_downstream_pb_routes(t_pb_routes& curr_pb_routes,
     }
 }
 
+static void rec_remove_downstream_pb_routes_ignore_used_items(t_pb_routes& curr_pb_routes,
```

Comment:
Add a declaration of this function at the top of the file in the local function section.
Also add a comment of what the function does above that declaration, and comment what each input is.

Reply:
We apply your comments.

Reply:
Thanks